common/scripted_triggers/wc_class_scripted_triggers.txt

```diff
@@ -1205,6 +1205,7 @@ allow_priest_trigger = {
 				OR = {
 					light_public_religion_trigger = yes
 					religion = cult_of_sunwell
+					religion = suntouched
 				}
```

Comment:
Consider adding it to `light_public_religion_trigger` and `light_true_religion_trigger`. It will force spirituals to have priest class.